pkg/minikube/reason/known_issues.go

```diff
@@ -1162,6 +1162,16 @@ var controlPlaneIssues = []match{
 		},
 		Regexp: re(`apiServer.certSANs: Invalid value`),
 	},
+	{
+		Kind: Kind{
+			ID:           "K8S_UNHEALTHY_CONTROL_PLANE,
```

Comment:
Missing a closing double quote

Reply:
ah thanks! I was using the online editor. Too lazy to create a branch.